src/frontend/use-user.tsx

```diff
@@ -63,7 +64,7 @@ export type UserProviderProps = React.PropsWithChildren<{ user?: UserProfile; pr
 /**
  * @ignore
  */
-const User = createContext<UserContext>({ isLoading: false });
+const User = createContext<UserContext>({} as UserContext);
```

Comment:
We can use the initial context to warn the user when they forget to include the `Provider`, eg

```js
const User = createContext<UserContext>({
  isLoading: false,
  checkSession: () => Promise.reject('You forgot to wrap your app in <UserProvider>')
});
```

We do the same in react https://github.com/auth0/auth0-react/blob/master/src/auth0-context.tsx#L141-L156